src/main/java/moe/nightfall/vic/integratedcircuits/gate/Gate7Segment.java

```diff
@@ -70,9 +72,11 @@
 	public static final int MODE_ANALOG = 1;
 	public static final int MODE_SHORT_SIGNED = 2;
 	public static final int MODE_SHORT_UNSIGNED = 3;
-	public static final int MODE_FLOAT = 4;
-	public static final int MODE_BINARY_STRING = 5;
-	public static final int MODE_MANUAL = 6;
+	public static final int MODE_INT_SIGNED = 4;
+	public static final int MODE_INT_UNSIGNED = 5;
+	public static final int MODE_FLOAT = 6;
+	public static final int MODE_BINARY_STRING = 7;
+	public static final int MODE_MANUAL = 8;
 
```

Comment:
You can't just change these, it would break backwards compatibility. You need to add int to the end as 7/8